api/v1alpha/api.proto

```diff
@@ -217,6 +217,9 @@ message PodFilter {
 
         // If not empty, the pods whose cgroup are listed will be returned.
         repeated string cgroups = 7;
+
+        // If not empty, the pods whose cgroup are a prefix of the listed cgroups will be returned.
+        repeated string cgroup_prefixes = 8;
```

Comment:
@sjpotter They are not actually prefixes... The pod's cgroup path is the prefix in this case, right?